frontend/src/modules/Environments/views/EnvironmentCreateForm.js

```diff
@@ -510,6 +514,7 @@ const EnvironmentCreateForm = (props) => {
                 notebooksEnabled: isModuleEnabled(ModuleNames.NOTEBOOKS),
                 mlStudiosEnabled: isModuleEnabled(ModuleNames.MLSTUDIO),
                 pipelinesEnabled: isModuleEnabled(ModuleNames.DATAPIPELINES),
+                omicsEnabled: isModuleEnabled(ModuleNames.OMICS),
```

Comment:
confirming no issues arise from not backfilling for this `environment.parameter` right?

Reply:
tested this and works as expected with default `Disabled` for existing environments